src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1703,7 +1703,16 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                         positionGetFunc: () => 0,
                         canReadFunc: () => true,
                         readAsyncFunc: (buffer, offset, count, cancellationToken) => syncFailure ? throw error : Task.Delay(1).ContinueWith<int>(_ => throw error)));
-                    Assert.Same(error, await Assert.ThrowsAsync<FormatException>(() => client.PostAsync(uri, content)));
+
+                    if (!UseManagedHandler && !PlatformDetection.IsUap)
```

Comment:
Nit: I have trouble with the double-negative and would prefer if this were inverted, e.g.
```C#
if (UseManagedHandler || IsUapHandler)
{
    HttpRequestException ...;
    Assert.Same(...);
}
else
{
    Assert.Same(...);
}
```